server.js

```diff
@@ -0,0 +1,37 @@
+const app = require('express')();
+const server = require('http').Server(app);
+const io = require('socket.io')(server);
+const next = require('next');
+
+const dev = process.env.NODE_ENV !== 'production';
+const port = parseInt(process.env.PORT, 10) || 3000;
```

Comment:
Don't think you need to use parseInt here, but a good safety measure.

Reply:
makes sense. Gonna remove!